network/p2p/gossip/gossip.go

```diff
@@ -139,4 +166,7 @@ func (g *Gossiper[T, U]) handleResponse(
 			continue
 		}
 	}
+
```

Comment:
would it be helpful to add time metrics here as well ?

Reply:
we could but I think these will be strongly correlated with either the bytes/n which is why i didn't add it

Reply:
we have some visibility w/ tracers but I think this is still nice to have. added.